src/common/github/pull-request-test-workflow.ts

```diff
@@ -37,7 +37,14 @@ export interface PullRequestTestOptions
    *
    * @default true
    */
-  readonly lighthouse?: boolean
+  readonly isLighthouseEnabled?: boolean
+
+  /**
+   * Setup Playwright end-to-end tests job.
+   *
+   * @default true
+   */
+  readonly isPlaywrightEnabled?: boolean
```

Comment:
@KrotkovDmitry @bdoof do you think we can already apply the modular approach since this PR? Or do we keep them separate?

Reply:
@gvidon  I guess I can start working under a modular approach for Playwright after this PR. As I can see [in this](https://github.com/ottofeller/templates/pull/290#discussion_r1230547736) discussion there are no any problems with it, but I didn't discuss it with @bdoof yet